ctms/crud.py

```diff
@@ -267,6 +268,86 @@ def get_contacts_by_any_id(
     return data
 
 
+def get_pending_acoustic_records(
+    db: Session,
+    end_time: datetime,
+):
+    """Get all the pending records up to a certain date."""
+    pending_records: List[PendingAcousticRecord] = (
+        db.query(PendingAcousticRecord)
+        .filter(
+            PendingAcousticRecord.update_timestamp < end_time,
+        )
+        .order_by(asc(Email.update_timestamp), asc(Email.email_id))
```

Comment:
Is this supposed to be ordered by ``PendingAcousticRecord.update_timestamp``, ``PendingAcousticRecord.email_id``?

Reply:
`PendingAcousticRecord.update_timestamp` and 
either `PendingAcousticRecord.email_id` or `PendingAcousticRecord.id`

nice catch, thanks